packages/ppx/src/css_to_emotion.re

```diff
@@ -168,39 +168,34 @@ and render_style_rule = (ident, rule: Style_rule.t): Parsetree.expression => {
   let dl_expr = render_declarations(block) |> Builder.pexp_array(~loc);
 
   let rec render_prelude_value = (s, (value, value_loc)) => {
+
     switch (value) {
     | Delim(":") => ":" ++ s
     | Delim(".") => "." ++ s
     | Delim(",") => ", " ++ s
     /* v can be ">", so we need an empty space between */
-    | Delim(v) => " " ++ v ++ " " ++ s
+    | Delim(v) => v ++ " " ++ s
```

Comment:
Do we have a  list of valid delimiters? Seems like here we have a few branches but are very poorly. Maybe we could have something similar as get_pseudoelement and have the entire list of them?

This would make the parsing more safe and better on error messaging.

Reply:
I agree, but if we have a list of valid delimiters i couldnt find it, i searched in the CSS spec and found this [page](https://www.w3.org/TR/selectors-4/#grammar), but i dont know if all the delimiters are there